custom-metrics-stackdriver-adapter/README.md

```diff
@@ -0,0 +1,113 @@
+# Custom Metrics - Stackdriver Adapter
+
+----
+
+Custom Metrics - Stackdriver Adapter is an implementation of [Custom Metrics
+API] based on Stackdriver per-pod metrics. It's purpose is to enable pod
+autoscaling based on Stackdriver custom metrics.
+
+----
+
+## Usage guide
+
+This guide shows how to set up Custom Metrics - Stackdriver Adapter and export
+metrics to Stackdriver in a compatible way. Once this is done, you can use
+them to scale your application, following [HPAv2 design].
+
+### Configure cluster
+
+1. Create Kubernetes cluster or use existing one, see [cluster setup].
+   Requirements:
+
+   * Kubernetes version 1.8.1 or newer running on GKE or GCE
+
+   * Monitoring scope `monitoring` set up on cluster nodes (see [OAuth 2.0 API
```

Comment:
AFAIK the scopes are present when you created a cluster with default settings, so maybe it's worth to write it in a less scary way?

Reply:
Done. I left most details and added some more, but added a comment that this is usually no-op. Alternatively I could just assume that if someone has disabled some scopes, he will now how to enable it back, WDYT?